README.md

```diff
@@ -1,5 +1,129 @@
+
+## Declarative UIs
+
+Data viewers/Editors based on shapes.
+
+- [shaperone](https://github.com/hypermedia-app/shaperone) <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/hypermedia-app/shaperone" align="top"> - SHACL Shapes Form generator; [playground](https://forms.hypermedia.app/playground); `MIT` license; `Typescript`.
+
+## IDE support
+
+- [Linked Data Extension](https://marketplace.visualstudio.com/items?itemName=Elsevier.linked-data) - VS Code Extension for editing RDF files with embedded SHACL validator and SPARQL engine.
+- [SHACL Language Server](https://marketplace.visualstudio.com/items?itemName=stardog-union.vscode-langserver-shacl) - A VS Code extension providing language intelligence (diagnostics, hover tooltips, auto-completion, etc.) for W3C standard SHACL via the Language Server Protocol.
+- [Mentor RDF for VS Code](https://marketplace.visualstudio.com/items?itemName=faubulous.mentor) - Code editing support for RDF, RDFS, OWL, SKOS, SHACL and SPARQL.
+
+## Books
+
+- [Validating RDF Data (2018)](https://book.validatingrdf.com)
+
+## Tutorials
+
+- [Shapes applications and tools - ISWC'20 Tutorial](https://www.validatingrdf.com/tutorial/iswc2020/)
+
+## Talks
+
+- [The Many Shapes of SHACL by Holger Knublauch](https://www.youtube.com/watch?v=ccs-KhnWR1U) - LOTICO Talk, 18 Jun 2020.
+- [One Ontology, One Data Set, Multiple Shapes with SHACL by Tara Raafat](https://www.youtube.com/watch?v=apG5K3zc4V0) - Connected Data London, 2019.
+- [An Overview of SHACL Shapes Constraint Language - Part 2](https://www.youtube.com/watch?v=TSDplfqw8rM) - TopQuadrant, 2017.
+- [An Overview of SHACL Shapes Constraint Language](https://www.youtube.com/watch?v=_i3zTeMyRzU) - TopQuadrant, 2017.
+
+## Presentations
+
+## Specifications
+
+- SHACL, W3C Recommendations & Notes
+  - [Shapes Constraint Language (SHACL)](https://www.w3.org/TR/shacl/) - W3C Recommendation, 20 July 2017.
+  - [SHACL Advanced Features](https://www.w3.org/TR/shacl-af/) - W3C Working Group Note, 08 June 2017.
+  - [SHACL JavaScript Extensions](https://www.w3.org/TR/shacl-js/) - W3C Working Group Note, 08 June 2017.
+  - [SHACL Test Suite and Implementation Report](https://w3c.github.io/data-shapes/data-shapes-test-suite/) - W3C Document 17 January 2024.
+  - [SHACL Use Cases and Requirements](https://www.w3.org/TR/shacl-ucr/) - W3C Working Group Note 20 July 2017.
+
+- SHACL, Community Group Latest Drafts & Notes
+  - [SHACL 1.2 Core](https://w3c.github.io/shacl/shacl-core/)
+  - [SHACL 1.2 SPARQL Extensions](https://w3c.github.io/shacl/shacl-sparql/)
+  - [SHACL Advanced Features 1.1](https://w3c.github.io/shacl/shacl-af/)
+  - [SHACL Compact Syntax](https://w3c.github.io/shacl/shacl-compact-syntax/)
+  - [SHACL JavaScript Extensions](https://w3c.github.io/shacl/shacl-js/)
+
```

Comment:
Maybe also include other SHACL-related specifications like the [The Shape Topologies algorithm](https://w3id.org/tree/specification/shape-topologies)?

Reply:
I propose a new subsection like "SHACL-related specifications" with all the random stuff here. Maybe we could also put [DASH Data Shapes](https://datashapes.org) here?

Reply:
Sure!

Reply:
It's in the last commit, already.